example-files/notebooks/mango-variants-viz.ipynb

```diff
@@ -0,0 +1,428 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Mango variant visualization tools \n",
+    "bdgenomics.mango.variantViz\n",
+    "\n",
+    "Note: where “population” is required as a parameters such as “AF_AFR” - allele frequency African/African American, see list of populations at: https://macarthurlab.org/2017/02/27/the-genome-aggregation-database-gnomad/\n",
+    "\n",
+    "### Load data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "from bdgenomics.adam.adamContext import ADAMContext\n",
+    "from bdgenomics.adam.rdd import AlignmentRecordRDD, CoverageRDD\n",
+    "ac = ADAMContext(spark)\n",
+    "data = ac.loadVariants(\"../../mango-core/src/test/resources/exome_chr22.adam\")\n",
+    "data_geno = ac.loadGenotypes(\"../../mango-core/src/test/resources/geno.test1.adam\")"
```

Comment:
do we link to core files in the other tests? Also, it may be better to use the original vcf files in case formats change

Reply:
we needs these new files because they have frequencies for multiple populations